File: testdata/erlang/sample.erl
-module(sample).
-export([start/0, greet/1]).

-include("records.hrl").
-include_lib("stdlib/include/assert.hrl").

%% Start the application.
start() ->
    io:format("Starting~n").

%% Greet a user.
greet(Name) ->
    io:format("Hello, ~s~n", [Name]).

%% Internal helper (not exported).
helper(X) ->
    X * 2.

File: testdata/julia/sample.jl
using LinearAlgebra
using Statistics

export greet, compute, Point

struct Point
    x::Float64
    y::Float64
end

function greet(name::String)
    println("Hello, $name")
end

function compute(a::Vector{Float64}, b::Vector{Float64})
    return dot(a, b)
end

function _internal(x)
    return x * 2
end
